packages/jsii/bin/jsii.ts

```diff
@@ -54,6 +54,12 @@ const warningTypes = Object.keys(enabledWarnings);
             desc: `List of warnings to silence (warnings: ${warningTypes.join(
               ',',
             )})`,
+          })
+          .option('strip-deprecated', {
```

Comment:
(Not necessary to do it in this PR)

What do you think about adding this option to the `jsii` component of `package.json` as well?

Reply:
I'm favorable to this but did not have time to go about it... Should be easy enough to do.